### PR TITLE
Add week1 homework files

### PR DESCRIPTION
**Pull request by gulyener**

### CODE REVIEW

File: Week1/homework/create-db-tables.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const sqlCommand = `
+  CREATE DATABASE new_world;
+
+  USE new_world; 
+
+  CREATE TABLE if not exists country(id INT AUTO_INCREMENT PRIMARY KEY, Name VARCHAR(50), Continent VARCHAR(50), Region VARCHAR(50), SurfaceArea FLOAT, IndepYear INT, Population INT, LifeExpectancy FLOAT, GNP FLOAT(7,2), GNPOld FLOAT(7,2), LocalName VARCHAR(50), GovernmentForm VARCHAR(50), HeadOfState VARCHAR(50), Capital VARCHAR(30));
+
+  CREATE TABLE if not exists city(id INT AUTO_INCREMENT PRIMARY KEY, Name VARCHAR(50), CountryCode VARCHAR(10), District VARCHAR(30), Population INT);
+`;
```
Comment (mogaleaf) [inline Week1/homework/create-db-tables.js]: Good for both id auto increment primary key ! Continent could have been an enum but varchar works also.

File: Week1/homework/insert-cities.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world',
+});
+
+connection.connect();
+
+const insertInto = 'INSERT INTO `city` (Name, CountryCode, District, Population) VALUES ?';
+
```
Comment (mogaleaf) [inline Week1/homework/insert-cities.js]: This is perfect !

File: Week1/homework/insert-cities.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  ['Haarlem', 'NLD', 'Noord-Holland', 156645],
+  ['Erlangen', 'DEU', 'Baijeri', 108336],
+  ['Leiden', 'NLD', 'Zuid-Holland', 121562],
+  ['Shanghai', 'CHN', 'Shanghai', 24240000],
+  ['Amman', 'JOR', 'Amman', 4008000],
+  ['Hadano', 'JPN', 'Kanagawa', 167378],
+];
+
+connection.query(insertInto, [cities], (error, results, fields) => {
+  if (error) {
```
Comment (mogaleaf) [inline Week1/homework/insert-cities.js]: I really like the way you are inserting the cities, it's neat !

File: Week1/homework/insert-countries.js [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    839,
+    'Malaysia',
+    'Constitutional Monarchy, Federation',
+    'Salahuddin Abdul Aziz Shah Alhaj',
+    'Kuala Lumpur',
+  ],
+];
+
+connection.query(insertInto, [countries], (error, results, fields) => {
+  if (error) {
```
Comment (mogaleaf) [inline Week1/homework/insert-countries.js]: Again really neat , I like it !

File: Week1/homework/make-queries.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+SELECT name FROM city WHERE CountryCode='NLD';
+
+SELECT population FROM city WHERE name='Rotterdam';
+
+SELECT name, SurfaceArea FROM country ORDER BY SurfaceArea DESC LIMIT 10;
+
+SELECT name, population FROM city ORDER BY population DESC LIMIT 10;
+
+SELECT SUM(population) FROM country;
+`;
```
Comment (mogaleaf) [inline Week1/homework/make-queries.js]: You queries are correct

**Comment by gulyener** [comment]
@mogaleaf Thanks for the review Cecile! I changed the continents' data type to enum.

**Pull request closed without merging**